### PR TITLE
Some bugfixes

### PR DESCRIPTION
**Pull request by Yoshax**

### CODE REVIEW

**Comment by SpadesNeil** [comment]
Thank you for fixing https://github.com/PolarisSS13/Polaris/issues/2311

File: html/changelogs/Yoshax-Bugfix.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (Anewbe) [inline html/changelogs/Yoshax-Bugfix.yml]: N3X15

Comment (SpadesNeil) [inline html/changelogs/Yoshax-Bugfix.yml]: tabeflip

**Comment by Anewbe** [comment]
> Long-Record-Goblins

I enjoy this concept.

**Pull request merged**